docs/blog/newsletter-february-2022.rst

```diff
@@ -0,0 +1,85 @@
+
+In the end, format is just one more choice to make when deciding how to present yourself to prospective employers. For an interesting take on how the pandemic has changed---or not changed---resumes, one writer suggested `The Pandemic Changed Everything About Work, Except the Humble Résumé <https://www.nytimes.com/2022/01/22/business/pandemic-work-resumes.html>`__ from The New York Times.
+
+---------------------------------------------
+What about the "tech" in "technical writing"?
+---------------------------------------------
+
+What technologies might a technical writer need to know? A recent discussion on this topic produced wide-ranging responses; and along the way, the conversation provided a glimpse into the range of industries and expectations our community works with.
+
+Write the Docs began as a community focused on developer documentation, with an emphasis on tools and workflows programmers are familiar with. But we've grown in breadth and depth since those days, and our notions of technologies we need to do our jobs have grown, too. Ideas about technologies important for documentarians fell into two main categories, with some replies emphasizing tools of the trade, and others focusing on domain knowledge -- the technologies you might be writing about.
+
+Overall, the consensus seemed to be "domain knowledge trumps all." When you're interviewing, show what you know about what the company builds -- the jargon and product and markets for security, for big data and machine learning, for devops tooling, or whatever it is that you'd be writing the docs for. Docs tools *might* matter to some interviewers, but they all *will* care about your familiarity with the company's product and market.
+
+As for docs tools, this has two sides: GUI-based vs tools in the docs-as-code category. For the first, helpful tools might be: MadCap Flare, Framemaker with and without XML schemas, SnagIt for screen captures. And a fair number of folks recommended tools for docs-as-code: Git, Markdown, static site generators, CI/CD pipelines, command-line tools, and possibly a scripting language. JavaScript was mentioned most often, but Python came in with quite a few votes too. 
+
+It's related to which code languages a technical writer might find most useful. Beyond JavaScript, Python and markup languages, there are plenty of other possibilities. JSON/YAML for data, XML for data and for content management, HTML/CSS because web development can be part of a tech writer's responsibilities, and SQL because there's nothing like it for learning how databases work.
```

Comment:
```suggestion
It's related to which code languages a technical writer might find most useful. Beyond JavaScript, Python, and markup languages, there are plenty of other possibilities. JSON/YAML for data, XML for data and for content management, HTML/CSS because web development can be part of a tech writer's responsibilities, and SQL because there's nothing like it for learning how databases work.
```